packages/web/src/common/store/wallet/sagas.ts

```diff
@@ -74,6 +77,8 @@ function* sendAsync({
   }
 
   try {
+    yield* call(optimisticallyDecreaseUserSolBalance, audioWeiAmount)
```

Comment:
I'm not as confident on the optimistic changes, maybe @DejayJD can weigh in

Also wonder if we still need optimism or if this was a remnant of eth sends

Reply:
i tried just invalidating the query cache, but there was a race condition - the refetch happens before the cache gets invalidated, so this still felt necessary

Reply:
I'm guessing it's because we need to wait to see the balance change on RPC before invalidating the cache